src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const button = document.getElementById('form-submit-button');
+
+button.addEventListener('click', event => {
+  event.preventDefault();
+});
```

Comment:
```suggestion
document.addEventListener('submit', event => {
  event.preventDefault();
});
```

It will fix form behaviour